monitor.py

```diff
@@ -140,9 +167,9 @@ def error_logs(error_type, s):
     last_t = last_sent[error_type]
     t = time.time()
 
-    # Send email only every 4 hours
+    # Send Slack notification only every 4 hours
     if t > last_t + 60 * 60 * 4:
-        send_email('%s [%d times]' % (error_type, n), s.split('\n'))
```

Comment:
I don't think we want to get rid of email.  There are some installations where people don't use slack